src/actions/team.js

```diff
@@ -50,7 +50,7 @@ export const fetchAllManagingTeams = (userId, managingTeams) => {
       allMembers = allMembers.concat(data[i].data);
     }
     console.log('allManagingTeams:', allManagingTeams);
```

Comment:
console.log removal in this line

Reply:
Thanks for reviewing. I have not removed all console.log instances. This is a big task and has to be achieved over time. But only those that I found while working on dashboard and timelog page performance.